unit_tests/compare_output.F90

```diff
@@ -0,0 +1,162 @@
+program compare_output
+
+
+use netcdf
+
+implicit none
+
+integer, parameter :: CRES=96
+integer, parameter :: ntiles=6
+integer, parameter :: npz=3
+integer, parameter :: nexpt=3
+integer, allocatable :: layout_x(:),layout_y(:)
+
+real, allocatable :: truth(:,:,:),expt(:,:,:),diff(:,:,:),accum_error(:)
+real, allocatable :: truth_3d(:,:,:,:),expt_3d(:,:,:,:),diff_3d(:,:,:,:)
+
+integer :: i1,i2,j1,j2,nx,ny,ierr1,ierr2,ierr3,i,j,k,ncid,varid,ncid2,t,t2,var
+
+character*240 :: baseline_path
+character*2   :: tile1,tile2
+character*1   :: lx,ly
+character*10  :: var_list_2d(4)
+character*10  :: var_list_3d(2)
+
+baseline_path='/scratch2/BMC/gsienkf/Philip.Pegion/stochastic_physics_unit_tests/baseline_20210806/'
```

Comment:
This may come in from environment variable in the future

Reply:
Yes,  I didn't. get to finish up the unit test part of this commit in the way I wanted since this also contained the CA fixes for decomposition, and that became a priority.